pkg/compiler/transformers/k8s/workflow.go

```diff
@@ -207,14 +207,20 @@ func BuildFlyteWorkflow(wfClosure *core.CompiledWorkflowClosure, inputs *core.Li
 		NodeDefaults: v1alpha1.NodeDefaults{Interruptible: interruptible},
 	}
 
-	obj.ObjectMeta.Name, obj.ObjectMeta.GenerateName, obj.ObjectMeta.Labels[ExecutionIDLabel], err =
-		generateName(wf.GetId(), executionID)
+	obj.ObjectMeta.Name, obj.ObjectMeta.GenerateName, obj.ObjectMeta.Labels[ExecutionIDLabel], obj.ObjectMeta.Labels[ProjectLabel],
```

Comment:
ohh man this is such a weird way of writing it. I so prefer capturing it into a local variable before assigning into a map

Reply:
i thought so too, was trying to stick to how it was originally written. done.